zerver/tests/test_subs.py

```diff
@@ -779,10 +779,10 @@ def test_set_stream_color(self):
         stream_name = sub['name']
         new_color = "#ffffff" # TODO: ensure that this is different from old_color
         result = self.client_post(
-            "/json/subscriptions/property",
+            "/api/v1/subscriptions/property",
             {"subscription_data": ujson.dumps([{"property": "color",
                                                 "stream": stream_name,
-                                                "value": "#ffffff"}])})
+                                                "value": "#ffffff"}])}, **self.api_auth(test_email))
```

Comment:
Can you put the `**self.api_auth(test_email)` on the next line in these really long ones?  It'll look better I think.